astropy/io/fits/tests/test_table.py

```diff
@@ -1239,7 +1240,7 @@ def test_assign_multiple_rows_to_table(self):
         assert tbhdu2.columns.columns[2].array[0] == ''
         assert (tbhdu2.columns.columns[3].array[0] ==
                 np.array([0., 0., 0., 0., 0.], dtype=np.float32)).all()
-        assert tbhdu2.columns.columns[4].array[0] == True  # noqa
```

Comment:
Again, this may not right, it may well have been important to test that the array holds things equivalent to a bool

Reply:
Fixed by 757d1d9fd8c8b55ade21f6dc2d566c9ca5e8047f